api/v1beta1/watcherapi_types.go

```diff
@@ -27,9 +27,14 @@ type WatcherAPISpec struct {
 	// Important: Run "make" to regenerate code after modifying this file
 
 	WatcherCommon `json:",inline"`
+
 	// +kubebuilder:validation:Required
 	// Secret containing all passwords / keys needed
 	Secret string `json:"secret"`
+
+	// +kubebuilder:validation:Required
+	// MemcachedInstance is the name of the Memcached CR that all watcher service will use.
+	MemcachedInstance string `json:"memcachedInstance"`
```

Comment:
Any reason to not set default value in WatcherAPI ?

Reply:
following the same patter as Secret, see coment above